src/components/Table/Table.jsx

```diff
@@ -568,7 +572,11 @@ const Table = props => {
       visibleData &&
       visibleData.length ? ( // don't show pagination row while loading
         <Pagination
-          {...view.pagination}
+          pageSize={paginationProps.pageSize}
+          pageSizes={paginationProps.pageSizes}
+          page={paginationProps.page}
+          isItemPerPageHidden={paginationProps.isItemPerPageHidden}
+          totalItems={paginationProps.totalItems < maxPages ? paginationProps.totalItems : maxPages}
```

Comment:
Could we cover this in a test? 1001 pages worth of data is given to the table, then assert the select contains only 1000 items.

Reply:
Done